components/organisms/PageWrapper/index.tsx

```diff
@@ -10,36 +10,20 @@ import {
 import { getGlobalData } from "@/lib/api/globals";
 import { GlobalDataProvider } from "@/contexts/GlobalData";
 import Header from "@/components/organisms/Header";
-import Footer from "@/components/global/Footer";
+import Footer from "@/components/organisms/Footer";
 import Center from "@rubin-epo/epo-react-lib/Center";
 
 const PageWrapper: FunctionComponent<
   PropsWithChildren<{ locale: string }>
 > = async ({ locale, children }) => {
   const globalData = await getGlobalData(locale);
-  const {
```

Comment:
these queries aren't really related to "global" data and are more specific to the footer